frontend/src/components/generic/CategoryChip.vue

```diff
@@ -39,3 +57,35 @@ export default {
   },
 }
 </script>
+
+<style lang="scss">
```

Comment:
Why not scoped style?

Reply:
Because dense works for normal chip as well. I'm not sure where to put these styles. Theoretically we would need to introduce a dense prop on v-chip itself. I could add another wrapper component around v-chip?

Reply:
If we want to introduce a dense property vor v-chip generally, then yes, better to create an e-chip which will wrap v-chip and get this new property.

Not sure, we need this at this stage, though. V-chip component is currently not used anywhere else (there's a .v-chip in css in Picasso.vue, but not sure what this is being used for).

The small or x-small property on the original v-chip doesn't do it for this case?

.e-category-chip: There's no reason this should be in the global namespace. We could also just use font-weight-medium, if there are no additional styles.

Reply:
Dense works different than small and x-small. The alignment to text is better and the paddings are relative to the size. I'll create a chip component.